doc/src/ForManagers.dox

```diff
@@ -55,17 +56,17 @@
  *   example, determining the correct frame range and colorspace of an
  *   image sequence.
  *
- * - The @ref Context supplied to a method can be used to understand
- *   the intentions of the caller, as well as to determine which part of
- *   an application is involved in the call, for example, which image read
- *   node in a compositor is resolving a reference.
+ * - The @ref Context supplied to a method can be used to understand the
+ *   intentions and/or requirements of the caller, as well as to
+ *   determine which part of an application is involved in the call,
+ *   this can be used to help determine the correct values for @ref
```

Comment:
[consider][minor]
```suggestion
 *   determine which part of an application is involved in the call.
 *   This can be used to help determine the correct values for @ref
```